dnd.sty

```diff
@@ -136,12 +136,12 @@
 \renewcommand{\labelitemi}{\raisebox{0.25ex}{\tiny{\( \bullet \)}}}
 
 % Fancy DnD 5e-style hline
-\renewcommand{\hline}{
+\renewcommand{\hline}{%
```

Comment:
`\hline` already has a function in LaTeX. Would it be possible to rename this to something like `\mline` followed by

```
\renewcommand{\hline}{% add deprecate code here
    \mline%
}
```